src/utils/trust/individual.trustee.mapper.ts

```diff
@@ -9,6 +9,13 @@ export const mapIndividualTrusteeToSession = (
   formData: Page.IndividualTrusteesFormCommon,
   trustee?: Trust.TrustIndividual
 ): Trust.IndividualTrustee => {
+  let stillInvolved = (formData.stillInvolved === "1") ? "Yes" : "No";
+
+  // If a boolean value isn't receieved from the web form (it could be null or undefined, e.g. if question not displayed), need to set null
+  if (formData.stillInvolved === null || formData.stillInvolved === undefined) {
+    stillInvolved = null as unknown as string;
```

Comment:
why force the type to string?

stillInvolved: string | null

Reply:
Because VS code is complaining about the types:

`Type 'null' is not assignable to type 'string'`

But will give you a call to discuss.

Reply:
Changed this.